src/forwarders.js

```diff
@@ -387,23 +387,30 @@ export default function Forwarders(mpInstance, kitBlocker) {
         }
     };
 
-    this.callSetUserAttributeOnForwarders = function(key, value) {
+    this.onHandleForwarderUserAttributes = function(key, value, functionName) {
```

Comment:
Probably a nitpick for me, but I think calling this `functionNameKey` or `functionNameAsKey` is more descriptive since we're using it as a lookup.